frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -402,4 +402,49 @@ describe("scenarios > question > filter", () => {
       cy.findByText("Rustic Paper Wallet"); // Product ID 1, Gizmo
     });
   });
+
+  it.skip("should not drop aggregated filters (metabase#11957)", () => {
+    const AGGREGATED_FILTER = "Count is less than or equal to 20";
+
+    cy.request("POST", "/api/card", {
+      name: "11957",
+      dataset_query: {
+        database: 1,
+        query: {
+          "source-query": {
+            "source-table": ORDERS_ID,
+            filter: [">", ["field-id", ORDERS.CREATED_AT], "2020-01-01"],
+            aggregation: [["count"]],
+            breakout: [
+              ["datetime-field", ["field-id", ORDERS.CREATED_AT], "day"],
+            ],
+          },
+          filter: ["<=", ["field-literal", "count", "type/Integer"], 20],
+        },
+        type: "query",
+      },
+      display: "table",
+      visualization_settings: {},
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.visit(`/question/${QUESTION_ID}`);
+    });
+
+    // Test shows two filter collapsed - click on number 2 to expand and show filter names
+    cy.get(".Icon-filter")
+      .parent()
+      .contains("2")
+      .click();
+
+    cy.findByText(AGGREGATED_FILTER);
+
+    cy.findByText(/^Created At is after/i)
+      .parent()
+      .find(".Icon-close")
+      .click();
```

Comment:
I was going for the simpler scenario (removing the "ordinary" filter). If you think I should include the second scenario as well (changing that filter's value), I can add it and run this test 2 times. @flamber

Reply:
It's probably fine like that.